src/app/static/src/app/components/projects/ShareProject.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <div>
+        <a @click="shareProject" href=""
+           v-translate="'share'"></a>
+        <modal :open="open">
```

Comment:
Don't know if it's worth worrying about, but this will repeat the modal for every link. Could cut down the bloat with v-if="open" on the modal?

Reply:
My sense is that since its only once per project, and most people will likely only have 1 project (or a small number), it makes more sense not to use conditional rendering.